upgrade/UPGRADE-v7.3.7.md

```diff
@@ -1,15 +1,15 @@
-# [Upgrade from v7.3.6 to v7.3.7-dev](https://github.com/shopsys/shopsys/compare/v7.3.6...7.3)
+# [Upgrade from v7.3.6 to v7.3.7](https://github.com/shopsys/shopsys/compare/v7.3.6...v7.3.7)
 
-This guide contains instructions to upgrade from version v7.3.6 to v7.3.7-dev.
+This guide contains instructions to upgrade from version v7.3.6 to v7.3.7.
 
 **Before you start, don't forget to take a look at [general instructions](https://github.com/shopsys/shopsys/blob/7.3/UPGRADE.md) about upgrading.**
 There you can find links to upgrade notes for other versions too.
 
 - update composer dependency `composer/composer` in your project ([#2314](https://github.com/shopsys/shopsys/pull/2314))
     - versions bellow `1.10.22` has [reported security issue](https://github.com/composer/composer/security/advisories/GHSA-h5h8-pc6h-jvvx)
-    - see #project-base-diff to update your project
+    - see [project-base-diff](https://github.com/shopsys/project-base/commit/462b5c612a52e199c6bbfafd71f4fdb6bb2e5548) to update your project
```

Comment:
Could you please add a following line to keep it simmilar like updating symfony/symfony
```
- run `composer update composer/composer`
```

Reply:
done :heavy_check_mark: